app/_locales/en/messages.json

```diff
@@ -1,4 +1,12 @@
 {
+  "CSS_loadingTakingTooLongActionText": {
```

Comment:
`CSS_` is my attempt at letting people know this is used in `CSS`. There is probably a better way?